arm/entrypoint.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+set -e
+
+# Create Resource group if it does not exists
+
+RESOURCE_GROUP_EXISTS=$(az group exists -n "$AZURE_RESOURCE_GROUP")
+if [[ $RESOURCE_GROUP_EXISTS == "false" ]]
+then
+  if [[ -z "$RESOURCE_GROUP_LOCATION" ]]
+  then
+    echo "RESOURCE_GROUP_LOCATION is not set."
+  else
+    az group create --name "$AZURE_RESOURCE_GROUP" --location "$RESOURCE_GROUP_LOCATION"
+  fi
+fi
+
+URI_REGEX="^(http://|https://)\\w+"
```

Comment:
\w will it work with more slashes or . etc?

Reply:
it wont match the whole url as / and . will be ignored. we can come up with a more robust url regex. the check will pass for anything with prefix http://|https:// followed by a character followed by any set of special | non special characters